src/app/components/account/account-avatar/account-avatar.tsx

```diff
@@ -8,18 +8,38 @@ interface AccountAvatarProps extends BoxProps {
   name: string;
 }
 export const AccountAvatar = memo(({ publicKey, name, ...props }: AccountAvatarProps) => {
+  let firstGrapheme: string = '';
+  if (name?.includes('Account')) {
+    // Displays the account number.
+    //
+    // Assumes the name of the account is something like "Account 4". This will
+    // probably break when the account has a BNS name with the string
+    // `"Account"` in it, although it's probably good enough for now.
+    firstGrapheme = name.split(' ')[1];
+  } else {
+    // Displays the first grapheme of the account name in uppercase.
+    //
+    // Using graphemes allows proper handling of emojis and other characters
+    // with multiple code points.
+    firstGrapheme = [...new Intl.Segmenter().segment(name)][0].segment;
+
+    // Will only transform some characters. E.g., emojis are unaffected.
+    firstGrapheme = firstGrapheme.toUpperCase();
+  }
+  const isFirstGraphemeEmoji = /\p{Extended_Pictographic}/u.test(firstGrapheme);
+
   const gradient = useAccountGradient(publicKey ?? '');
 
-  const circleText = name?.includes('Account') ? name.split(' ')[1] : name[0];
   return (
     <Circle
       flexShrink={0}
-      backgroundImage={gradient}
-      color={color('bg')}
+      background={isFirstGraphemeEmoji ? '#f4f4f6' : color('bg')}
```

Comment:
@kyranjamie @fbwoolf what's the best way to handle this "magic" color hex? Extract to variable? Do we have a color palette?

Reply:
Some colours are defined in the stacks ui package, but we don't have a set practice for handling hex codes. Definite room for improvement here, can discuss in next weeks' component call.

Reply:
Alright, will leave in place for now until we come up with something better.